databuilder/justfile

```diff
@@ -0,0 +1,118 @@
+# make a new virtualenv so we don't use the main docs one, which is why we're
+# not using $VIRTUAL_ENV here.
+export VIRTUAL_ENV  := ".venv"
+
+# TODO: make it /scripts on windows?
+export BIN := VIRTUAL_ENV + "/bin"
+export PIP := BIN + "/python -m pip"
+# enforce our chosen pip compile flags
+export COMPILE := BIN + "/pip-compile --allow-unsafe --generate-hashes"
+
+
+# list available commands
+default:
+    @{{ just_executable() }} --list
+
+
+# clean up temporary files
+clean:
+    rm -rf .venv
+
+
+# ensure valid virtualenv
+virtualenv:
+    #!/usr/bin/env bash
+    PYTHON_VERSION=$(which python3.10)
```

Comment:
note: This caught me out because I don't have Python 3.10 installed. But not a big deal.